package/setup.py

```diff
@@ -55,7 +55,7 @@
             "sphinx~=5.3.0",
             "sphinx_copybutton==0.3.1",
             "sphinx-notfound-page",
-            "sphinx_rtd_theme==1.2.0",
+            "sphinx_rtd_theme==1.3.0",
```

Comment:
That's not exactly what I expected though 🤔 Maybe it's because the dependency is pinned? Maybe it's because we should use `increase` rather than `increase-if-necessary`? All speculation here.

Reply:
Yes, I think it is because we pinned the requirements. Looking at the [versioning startegy](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#versioning-strategy) action, even `increase` won't help. `widen` would have been great but pip does not support it. May be we should provide a range for the dependencies and use `increase-if-necessary` ?


Option | Action
-- | --
increase | Always increase the minimum version requirement to match the new version. If a range already exists, typically this only increases the lower bound.
increase-if-necessary | Leave the constraint if the original constraint allows the new version, otherwise, bump the constraint.

Reply:
Or maybe dev dependencies can stay pinned. No strong opinion on this one, it only affects Viz itself